Cargo.toml

```diff
@@ -29,3 +29,6 @@ tonic = { version = "0.12", features = [
     "codegen",
     "prost",
 ] }
+
+[patch.crates-io]
+datafusion-federation = { git = "https://github.com/nathanielc/datafusion-federation", branch = "chore/datafusion-45" }
```

Comment:
Will remove this once https://github.com/datafusion-contrib/datafusion-federation/pull/112 merges